i18n/zh/docusaurus-plugin-content-docs/current/userguide/roleseparation/federated-resource-quota.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Federated ResourceQuota
```

Comment:
Looks like the following is the same as above. So I skip it. Please incorporate the changes above to the following.